06_puppy/sahidakhtar1/main_test.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutPut(t *testing.T) {
```

Comment:
This is not testing anything. Remove.

Reply:
Just added one line out put test to achieve 100% coverage.

Reply:
This test still isn't right, because it is not testing that the output of `main()` is correct, just a small part of it. `main()` should be changed to use `fmt.Fprint*(out, ...)` calls so all output can be captured by this test case.

Furthermore, you should not be creating mapStores, puppies or anything else in this test function because they are not the things this test is for. This test is for `main()` so it should only call `main()` verify that it performed correctly.